### PR TITLE
Add Data Path Operation feature page

### PR DESCRIPTION
**Pull request by rollandf**
Changes proposed in this pull request:

- Add Feature Page for  Data Path Operations

I confirm that this pull request was submitted according to the [contribution guidelines](/CONTRIBUTING.md): @rollandf

This pull request needs review by: @mureinik @nirs @laravot

### CODE REVIEW

**Comment by rollandf** [comment]
@aglitke

**Comment by mykaul** [comment]
@rollandf - who can review it so we'll merge it?

File: source/develop/release-management/features/storage/data-path-operations.html.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+*   Liron Aravot (<laravot@redhat.com>)
+*   Nir Soffer (<nsoffer@redhat.com>)
+*   Fred Rolland (<frolland@redhat.com>)
+
+### Detailed Description
+
+There are a lot of data path operations in oVirt. For example: move disk, create VM from template, create template from VM, live storage migration and more.
+These operations are performed on the SPM host, potentially creating a bottleneck in scale setups or if the operations are performed frequently.
+
+By adding a new verb 'SDM.copy_data' available on all hosts, the engine is capable to convert the data path operations flows to run on any host.
```
Comment (aglitke) [inline source/develop/release-management/features/storage/data-path-operations.html.md]: I think this 'feature' is about the infrastructure that enables datapath operations on non-SPM hosts.  SDM.copy_data is just one example verb which uses the infrastructure.  We also have SDM.merge and SDM.amend_volume with many more planned.

File: source/develop/release-management/features/storage/data-path-operations.html.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+    -   description: The destination endpoint
+        name: destination
+        type: *CopyDataEndpoint
+```
+
+### Engine
+
+The engine will monitor the operation using the `Job` API.
+In case the job is not present, the status of the operation will determined by the lease status on the volume information retrieved from VDSM.
```
Comment (aglitke) [inline source/develop/release-management/features/storage/data-path-operations.html.md]: We examine the lease state to determine if a copy may still be active.  If the lease is free we can know whether the copy completed by checking the volume legality.

File: source/develop/release-management/features/storage/data-path-operations.html.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+- CopyImageGroupVolumesDataCommand
+- CopyDataCommand
+
+In addition, the framework for tracking progress on entities is added to the infrastructure.
+
+### Testing
+
+- Move a disk between storage domains. Check that the operation may run on none-SPM hosts.
+- Create a VM from template with clone provisioning.  Check that the operation may run on none-SPM hosts.
+
```
Comment (aglitke) [inline source/develop/release-management/features/storage/data-path-operations.html.md]: We should add information about testing negative flows.  For example, take down the network of a vdsm host after the Job has started.  Make sure engine can recover when the host comes back (check if the job failed or succeeded).  Make sure engine can recover if the host never comes back.

**Comment by rollandf** [comment]
@aglitke I addressed your comments. Please see if it is OK.
